addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -302,15 +302,11 @@ var ListRenderer = BasicRenderer.extend({
      * @returns {jQuery} a <button> element
      */
     _renderButton: function (record, node) {
-        var $button = $('<button>', {
-            type: 'button',
-            title: node.attrs.string,
-        });
+        var $button = this._renderButtonFromNode(node);
         if (node.attrs.icon) {
             $button.addClass('o_icon_button');
-            $button.append($('<i>', {class: 'fa ' + node.attrs.icon}));
-        } else {
-            $button.text(node.attrs.string);
+            var $i = $button.children().detach();
+            $button.empty().append($i);
```

Comment:
why?

Reply:
To do the equivalent as before (icon OR text for list buttons, the icon buttons in lists have no button appearance thanks to the `o_icon_button` class).

Reply:
i don't like that :D

Reply:
Meh, I will parametrize the `_renderButtonFromNode` then